app/src/main/java/com/example/capsule/ui/outfitSuggestion/OutfitSuggestionFragment.kt

```diff
@@ -195,14 +213,24 @@ class OutfitSuggestionFragment: Fragment(), LocationListener {
                     isSuggested = true
                 )
                 outfitSuggestionViewModel.insert(entry)
+                clothingInserted = true
+            }
+            if (clothingInserted) {
+                findNavController().navigate(R.id.action_navigation_outfits_suggestion_to_navigation_outfits_history)
+                with(requireActivity().getPreferences(Context.MODE_PRIVATE).edit()) {
+                    putLong(getString(R.string.has_inserted_for_day_key), calendar.timeInMillis)
+                    apply()
+                }
+                Toast.makeText(requireActivity(), R.string.empty_outfit_log, Toast.LENGTH_SHORT).show()
```

Comment:
The string should be `R.string.outfit_logged` for this case right?

Reply:
Oh that's right.